### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by llopen-sourcell**
Before submitting your pull request, check whether your code adheres to PHPMailer
coding standards by running the following command:

`./vendor/bin/php-cs-fixer --diff --dry-run --verbose fix `

And committing eventual changes. It's important that this command uses the specific version of php-cs-fixer configured for PHPMailer, so run `composer install` within the PHPMailer folder to use the exact version it needs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ See [changelog](changelog.md).
 - Became an Apache incubator project on Google Code in 2010, managed by Jim Jagielski.
 - Marcus created his fork on [GitHub](https://github.com/Synchro/PHPMailer) in 2008.
 - Jim and Marcus decide to join forces and use GitHub as the canonical and official repo for PHPMailer in 2013.
-- PHPMailer moves to the [PHPMailer organisation](https://github.com/PHPMailer) on GitHub in 2013.
+- PHPMailer moves to the [PHPMailer organization](https://github.com/PHPMailer) on GitHub in 2013.
```
Comment (Synchro) [inline README.md]: Americanism

File: README.md [hunk 2]
```diff
@@ -184,20 +184,20 @@ We're particularly interested in fixing edge-cases, expanding test coverage and
...
+Please *don't* use the SourceForge or Google Code projects anymore; they are obsolete and no longer maintained.
 
 ## Sponsorship
 Development time and resources for PHPMailer are provided by [Smartmessages.net](https://info.smartmessages.net/), a powerful email marketing system.
 
 <a href="https://info.smartmessages.net/"><img src="https://www.smartmessages.net/img/smartmessages-logo.svg" width="250" height="28" alt="Smartmessages email marketing"></a>
 
-Other contributions are gladly received, whether in beer 🍺, T-shirts 👕, Amazon wishlist raids, or cold, hard cash 💰. If you'd like to donate to say "thank you" to maintainers or contributors, please contact them through individual profile pages via [the contributors page](https://github.com/PHPMailer/PHPMailer/graphs/contributors).
+Other contributions are gladly received, whether in beer 🍺, T-shirts 👕, Amazon wishlist raids, or cold, hard cash 💰. If you'd like to donate to say "thank you" to maintainers or contributors, please contact them through individual profile pages via [the contributor's page](https://github.com/PHPMailer/PHPMailer/graphs/contributors).
```
Comment (Synchro) [inline README.md]: It's plural, not possessive

File: README.md [hunk 3]
```diff
@@ -184,20 +184,20 @@ We're particularly interested in fixing edge-cases, expanding test coverage and
...
 
-If you have git clones from prior to the move to the PHPMailer GitHub organisation, you'll need to update any remote URLs referencing the old GitHub location with a command like this from within your clone:
+If you have git clones from prior to the move to the PHPMailer GitHub organization, you'll need to update any remote URLs referencing the old GitHub location with a command like this from within your clone:
 
 ```sh
 git remote set-url upstream https://github.com/PHPMailer/PHPMailer.git
 ```
 
-Please *don't* use the SourceForge or Google Code projects any more; they are obsolete and no longer maintained.
+Please *don't* use the SourceForge or Google Code projects anymore; they are obsolete and no longer maintained.
```
Comment (Synchro) [inline README.md]: Americanism

File: README.md [hunk 4]
```diff
@@ -184,20 +184,20 @@ We're particularly interested in fixing edge-cases, expanding test coverage and
 
 If you found a mistake in the docs, or want to add something, go ahead and amend the wiki - anyone can edit it.
 
-If you have git clones from prior to the move to the PHPMailer GitHub organisation, you'll need to update any remote URLs referencing the old GitHub location with a command like this from within your clone:
+If you have git clones from prior to the move to the PHPMailer GitHub organization, you'll need to update any remote URLs referencing the old GitHub location with a command like this from within your clone:
```
Comment (Synchro) [inline README.md]: Americanism

File: README.md [hunk 5]
```diff
@@ -130,7 +130,7 @@ try {
 }
 ```
 
-You'll find plenty more to play with in the [examples](https://github.com/PHPMailer/PHPMailer/tree/master/examples) folder.
+You'll find plenty more to play within the [examples](https://github.com/PHPMailer/PHPMailer/tree/master/examples) folder.
```
Comment (Synchro) [inline README.md]: That's not a correction!

File: README.md [hunk 6]
```diff
@@ -171,7 +171,7 @@ Please disclose any vulnerabilities found responsibly - report any security prob
...
 
-PHPMailer versions prior to 5.2.20 (released December 28th 2016) are vulnerable to [CVE-2016-10045](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-10045) a remote code execution vulnerability, responsibly reported by [Dawid Golunski](https://legalhackers.com/advisories/PHPMailer-Exploit-Remote-Code-Exec-CVE-2016-10045-Vuln-Patch-Bypass.html), and patched by Paul Buonopane (@Zenexer).
+PHPMailer versions prior to 5.2.20 (released December 28th, 2016) are vulnerable to [CVE-2016-10045](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-10045) a remote code execution vulnerability, responsibly reported by [Dawid Golunski](https://legalhackers.com/advisories/PHPMailer-Exploit-Remote-Code-Exec-CVE-2016-10045-Vuln-Patch-Bypass.html), and patched by Paul Buonopane (@Zenexer).
```
Comment (Synchro) [inline README.md]: If you're going to change that one, probably best to change all the other dates in this doc (or at least in that section) in the same way
